angular/core/components/group_page/description_card/description_card.haml

```diff
@@ -0,0 +1,12 @@
+%section.description-card{aria-labelledby: "description-card-title"}
+  %h2#description-card-title.lmo-card-heading{translate: "group_page.description"}
+  %div{ng-hide: "editorEnabled"}
+    .description-card__placeholder.lmo-placeholder.lmo-hint-text{translate: "group_page.description_placeholder", ng-if: "showDescriptionPlaceholder()"}
+    .description-card__text.lmo-last.lmo-markdown-wrapper{marked: "group.description"}
+    %button.description-card__edit.lmo-link{ng-click: "enableEditor()", translate: "common.action.edit"}
+  %div{ng-show: "editorEnabled"}
+    .lmo-form-group
+      .lmo-relative.lmo-textarea-wrapper
+        %textarea.lmo-textarea.description-card__textarea.form-control.lmo-primary-form-input{msd-elastic: "true", ng-model: "editableDescription", ng-model-options: "{debounce: 150}"}
+    %button.description-card__save.lmo-link{ng-click: "save()", translate: "common.action.save"}
```

Comment:
Ok, I did it, ha. Lemme know what you think :)

![group-description-edit-link-2](https://cloud.githubusercontent.com/assets/2882097/20587559/7c25e7ae-b273-11e6-92e0-cf3968186bf2.gif)